### ISSUE TITLE
Versioning confusion in Readme

### ISSUE THREAD

**Issue by itpdg** [rust-ndarray/ndarray#950]
Dear all,

Tl;dr: In the Readme.md dependencies-example, there is ndarray 0.14 together with blas-src 0.7, while according to the table below it should actually be 0.14 with 0.6.1 (or 0.15 with 0.7).

I hoped it is ok that I opened an issue for such a (seemingly) minor problem. But here's the thing: I was trying to get a simple program running, basically only matrix multiplication:

```rust
extern crate ndarray;
use ndarray::prelude::*;

fn main() {
	let	a_id_mat: Array2<f64> = 2. * Array::eye(3);

	let x = array![[1.,2.,3.]];
	let x = x.t();
	let y = a_id_mat.dot(&x);
	
	println!("x={}",x);
	println!("y={}",y);
}
```
with a simple Cargo-file, as suggested in the Readme:
```
[package]
name = "nda_mm"
version = "0.1.0"
authors = ["M. M."]
edition = "2018"

[dependencies]
ndarray = { version = "0.14.0", features = ["blas"] }
blas-src = { version = "0.7", default-features = false, features = ["openblas"] }
openblas-src = { version = "0.9", default-features = false, features = ["cblas", "system"] }
```
As warned in the Readme, this won't link correctly (basically, I get `undefined reference to cblas_dgemm`), at least on Debian stable.

(Actually, I would like to ask how the colliding versions are handled with ndarray-linalg, but that's maybe not the right place).

Kind regards and thanks for creating ndarray :)

**Comment by bluss** [comment]
Hey, I think this would be solved when fixing all version numbers for the 0.15 release.
